samples/azure-quantum/simulators/SparseSimulator.ipynb

```diff
@@ -0,0 +1,2274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "acquired-exhibit",
+   "metadata": {},
+   "source": [
+    "# Sparse Simulator\n",
+    "\n",
+    "The sparse simulator is a simulator that utilizes a sparse representation of quantum state vectors, as opposed to the full state simulator. This feature allows the sparse simulator to minimize the memory footprint used to represent quantum states that are sparse in the computational basis, thus enabling simulations over a larger number of qubits. The sparse simulator is efficient for a class of quantum algorithms with a smaller number of states in superposition. This simulator is an unlocker for users to explore larger applications than what can be explored using the full state simulator alone. For more information about the sparse simulator see [this paper](https://arxiv.org/abs/2105.01533).\n",
+    "\n",
+    "Let's see the advantages of the sparse simulator over the [full state simulator](https://docs.microsoft.com/en-us/azure/quantum/user-guide/machines/full-state-simulator).\n",
```

Comment:
```suggestion
    "In this sample, we'll look at the advantages of the sparse simulator over the [full-state simulator](https://docs.microsoft.com/azure/quantum/user-guide/machines/full-state-simulator).\n",
```

Please note that `en-us` in docs URLs overrides users' locale preferences and should always be removed from links.

Reply:
We should be consistent, to be sure. "Full state" strikes me as incorrect grammatically, but at the same time I don't think we should go and fix everywhere else that occurs at the moment. @SoniaLopezBravo, do you have any advice here from a docs perspective?

Reply:
For now I'm leaving dashes (`full-state`).
And on my side I'll try to clarify with the rest of the team.

Reply:
It still seems really wrong to me, but if we're using "full state" consistently elsewhere, we should do so here as well.

Reply:
Changed to "full state".